Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -89,6 +93,13 @@ script:
             return sendRequest('PUT',args.uri, args.body);
         case 'demisto-api-delete':
             return sendRequest('DELETE',args.uri);
+        case 'demisto-api-multipart':
+            return sendMultipart(args.uri, args.entryID, args.body);
+        case 'demisto-api-download':
+            var res = sendRequest('GET',args.uri,args.body,true);
+            var filename = (args.filename) ? args.filename : (res.Headers['Content-Disposition']) ? res.Headers['Content-Disposition'][0].split('=')[1] : res.Path;
```

Comment:
I would check the split returned more than one element (and do this in if else...)